data_adapter_oemof/calculations.py

```diff
@@ -1,9 +1,31 @@
+import warnings
+
 from oemof.tools.economics import annuity
 
 
+def calculation(func):
+    r"""
+    This is a decorator that allows calculations to fail
+    """
+
+    def decorated_func(*args, **kwargs):
+        try:
+            func(*args, **kwargs)
```

Comment:
should return func(*args, **kwargs) otherwise decorated_func (and therefore calculation) returns None if calculation is successful

Reply:
Done in [86c60f6](https://github.com/sedos-project/data_adapter_oemof/pull/5/commits/86c60f6eb7be7d8da033e294d0fd75b1b68266f0)